compute_tools/src/http/openapi_spec.yaml

```diff
@@ -68,6 +68,45 @@ paths:
               schema:
                 $ref: "#/components/schemas/Info"
 
+  /schema/objects:
+    get:
+      tags:
+        - Info
+      summary: Get databases and roles in the schema.
+      description: ""
+      operationId: getSchemaObjects
+      responses:
+        200:
+          description: Compute schema objects
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/SchemaObjects"
+
+  /schema/ddl:
+    get:
+      tags:
+        - Info
+      summary: Get schema dump
+      parameters:
+        - name: db_name
```

Comment:
Would it be more consistent to just use `database` or do what Postgres does and use `dbname`?

Reply:
done, db_name -> database